# Message for a station with name, pose, degrees, and id
string name
float32 positionX
float32 positionY
float32 positionZ
float32 orientationX
float32 orientationY
float32 orientationZ
float32 orientationW
string id

